lib/components/ConflictPicker.vue

```diff
@@ -194,7 +194,7 @@ export default defineComponent({
 		},
 
 		skipButtonLabel() {
-			return n('Skip this file', 'Skip {count} files', this.conflicts.length, { count: this.conflicts.length })
+			return n('Skip {count} file', 'Skip {count} files', this.conflicts.length, { count: this.conflicts.length })
```

Comment:
Or to keep the current 1 shortcut

```suggestion
			if (this.conflicts.length === 1) {
				return t('Skip this file')
			}
			return n('Skip {count} file', 'Skip {count} files', this.conflicts.length, { count: this.conflicts.length })
```